bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -416,7 +416,7 @@ func (r *KubeadmConfigReconciler) handleClusterNotInitialized(ctx context.Contex
 	if scope.Config.Spec.InitConfiguration == nil {
 		scope.Config.Spec.InitConfiguration = &bootstrapv1.InitConfiguration{
 			TypeMeta: metav1.TypeMeta{
-				APIVersion: "kubeadm.k8s.io/v1beta1",
+				APIVersion: "kubeadm.k8s.io/v1beta3",
```

Comment:
@fabriziopandini  I guess this wasn't a problem because this code was never actually used?

Reply:
I need to dig into this, I will follow up ASAP (eventually we can drop this from this PR if you want to move forward)

Reply:
I think the PR doesn't change it for the worse